wordclock_interfaces/templates/app.html

```diff
@@ -82,6 +82,128 @@
                                                 single-line menu-props="bottom">
                                             </v-select>
                                         </v-card-text>
+                                        
+                          <v-divider></v-divider>
```

Comment:
When I saw the new web interface, I was wondering to maybe even move it to a new tab: Besides "Control" and "Colors" maybe something like "Text", "Message" or similar. How do you think about that, @FrankX0

Reply:
That was my first idea. But I think it would take too much space.
With the collapsible approach in "Control", I think it fits in nicely.